server/app.ts

```diff
@@ -1,30 +1,30 @@
 /* istanbul ignore file */
 
-import path from 'path'
-import express from 'express'
 import flash from 'connect-flash'
+import express from 'express'
+import path from 'path'
 
 import createError from 'http-errors'
 import methodOverride from 'method-override'
 
-import nunjucksSetup from './utils/nunjucksSetup'
 import errorHandler from './errorHandler'
 import authorisationMiddleware from './middleware/authorisationMiddleware'
 import { metricsMiddleware } from './monitoring/metricsApp'
+import nunjucksSetup from './utils/nunjucksSetup'
```

Comment:
Nice, I'd be up for introducing alphabetical sorting if the rest of the team didn't object, used to do it on previous project and found it easier to parse code